views/layout.hbs

```diff
@@ -8,7 +8,9 @@
     <title>{{title}}</title>
     <link rel="stylesheet" href="/stylesheets/style.css" />
   </head>
-
+  <nav>
+    <a href="/drones">All Drones</a>
+  </nav>
   <body>
```

Comment:
It is not a good practice to have html tags outside the `body` tag.
Latest versions of html allow it, but with older versions, browser would ignore and any html outside the body tag

My recommendations - stick to conventions :)

```
The body tag is like a wrapper. And everything that you visibly see on the screen goes 
inside or between the body tags. If you wanted to put a line of text outside of the body, 
the browser will ignore this and will not  treat it properly as HTML, since that statement 
would be invalid and incorrect. May 13, 2019
```